pkg/config.go

```diff
@@ -22,23 +26,33 @@ func (c *Client) getConfigMap(namespace, name string) (configMap *ConfigMap, err
 	return
 }
 
+// GetSystemConfig loads various system configurations and bundles them into a map.
+// The configuration is cached once it is loaded, and that cached value is used from here on out.
 func (c *Client) GetSystemConfig() (config map[string]string, err error) {
+	if c.systemConfig != nil {
+		return c.systemConfig, nil
+	}
+
 	namespace := "onepanel"
-	configMap, err := c.getConfigMap(namespace, "onepanel")
-	if err != nil {
+	configMap, configMapErr := c.getConfigMap(namespace, "onepanel")
```

Comment:
why not just use `err`?

Reply:
Originally I got an err shadowed compilation error. 
But I'm not getting it now - pushing up a fix.